### PR TITLE
issue #1030 is addressed please check it.

### LINKED ISSUE #1030: `docker-compose.minimal` not using OCB
`docker-compose.minimal` is not using the OCB.
https://github.com/open-telemetry/opentelemetry-demo/blob/main/docker-compose.minimal.yml#L504

We need to update it accordingly.

Just a copy and paste job:
https://github.com/open-telemetry/opentelemetry-demo/blob/main/docker-compose.yml#L647-L672

### PR DESCRIPTION
**Pull request by VAISHALI-NILE**
Fix #1030

### CODE REVIEW

File: docker-compose.minimal.yml [hunk 1]
```diff
@@ -500,9 +500,14 @@ services:
     logging: *logging
 
   # OpenTelemetry Collector
-  otelcol:
-    image: otel/opentelemetry-collector-contrib:0.76.1
+    otelcol:
```
Comment (julianocosta89) [inline docker-compose.minimal.yml:503]:
```suggestion
  otelcol:
```

File: docker-compose.minimal.yml [hunk 2]
```diff
@@ -521,7 +526,6 @@ services:
     depends_on:
       - jaeger
     logging: *logging
-
```
Comment (julianocosta89) [inline docker-compose.minimal.yml:524]: Could you also add the line break back here?

**Review by julianocosta89 — APPROVED** [review]
LGTM

File: docker-compose.minimal.yml (hunk 2 above)
Comment (VAISHALI-NILE) [reply docker-compose.minimal.yml:524]: sure

File: docker-compose.minimal.yml [hunk 3]
```diff
@@ -521,7 +526,7 @@ services:
...
+
```
Comment (julianocosta89) [inline docker-compose.minimal.yml:529]: The new line shouldn't have spaces

Comment (VAISHALI-NILE) [reply docker-compose.minimal.yml:529]: ok

**Review by mviitane — APPROVED** [review]
LGTM. This is now aligned with the main docker-compose file.

**Pull request merged**